citrination/commands/create_dataset.py

```diff
@@ -17,9 +19,12 @@ def main(self):
     def setup(self):
         """Setup this application."""
         cli.app.CommandLineApp.setup(self)
-        self.add_param("-k", "--api_key", help="Citrination API key.", required=True)
-        self.add_param("-e", "--citrination_host", help="Citrination host.",
-                       default='https://citrination.com', required=False)
+        self.add_param("-p", "--project", default=None, required=False,
```

Comment:
Hmm, I like the idea of just being able to set your project name that you want to connect to. What about also allowing a --host option. Then we would try to connect in this order:

1. If host is set use that
2. If project is set, then connect to https://project.citrination.com
3. If neither is set, then connect to https://citrination.com